src/panels/lovelace/types.ts

```diff
@@ -38,7 +38,7 @@ export interface LovelaceBadge extends HTMLElement {
 export interface LovelaceCard extends HTMLElement {
   hass?: HomeAssistant;
   isPanel?: boolean;
-  editMode?: boolean;
+  editMode?: any;
```

Comment:
Let's go ahead and type this to 

```suggestion
  editMode?: boolean | EditMode;
```

```ts
interface EditMode {
	selectedRow: Number;
	selectedCard: Number;
}
```

Reply:
I want to limit it as little as possible. Custom cards should be able to use this too and populate with whatever data they want.

Reply:
```ts
interface EditMode {
    selectedRow: Number;
    selectedCard: Number;
    [key: string]: any;
}
```

Reply:
or we should create a part of the edit mode that can be any

```ts
interface EditMode {
    selectedRow: Number;
    selectedCard: Number;
    data: any; // But not data as the key
}

```